src/clndr.js

```diff
@@ -1003,21 +1001,9 @@
             if (this.options.events) {
                 // Are any of the events happening today?
                 if (this.options.multiDayEvents) {
+                    var targetEndDate = target.date.clone().endOf("day");
                     filterFn = function () {
-                        var isSameStart = target.date.isSame(
-                            this._clndrStartDateObject,
-                            'day');
-                        var isAfterStart = target.date.isAfter(
-                            this._clndrStartDateObject,
-                            'day');
-                        var isSameEnd = target.date.isSame(
-                            this._clndrEndDateObject,
-                            'day');
-                        var isBeforeEnd = target.date.isBefore(
-                            this._clndrEndDateObject,
-                            'day');
-                        return (isSameStart || isAfterStart)
-                            && (isSameEnd || isBeforeEnd);
+                        return (this._clndrStartDateObject <= targetEndDate && target.date <= this._clndrEndDateObject);
```

Comment:
n.b. comment should be overly explicit:

> If the event date has the same start date or is after the start date of the interval, and it has either the same end date or is before the end date.

Reply:
This also looks good to go. I'm going to bring this into a separate branch once #340 is merged into master.